core/controllers/admin.py

```diff
@@ -107,11 +107,13 @@ class AdminPage(
     URL_PATH_ARGS_SCHEMAS: Dict[str, str] = {}
     HANDLER_ARGS_SCHEMAS: Dict[str, Dict[str, str]] = {'GET': {}}
 
-    @acl_decorators.can_access_admin_page
-    def get(self) -> None:
+    # Here we use type Any because the sub-classes of 'Basehandler' can have
```

Comment:
I don't understand why you need Any here when you didn't previously?

Reply:
There is a check that fails if the subclass changes the method signature from that of the parent class. My guess is that this CI check only looks at files that were edited in the PR and this file is pretty old. Since I edited it now, it fell under the purview of the check and had to be updated.

Reply:
Hmm, that seems off to me. All CI checks are stateless -- they run on the entirety of the codebase and they don't look at changes from the previous code, so something seems off about "this CI check only looks at files that were edited in the PR". Could you point me to the check in question?

Reply:
not relevant now since I've deleted the admin page handler entirely but I'll check this separately